services/movie.py

```diff
@@ -0,0 +1,47 @@
+import init_django_orm  # noqa: F401
+
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet[Movie]:
+
+    if not genres_ids and not actors_ids:
+        return Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        return Movie.objects.filter(
+            genres__id__in=genres_ids
+        ).filter(actors__id__in=actors_ids)
+
+    if genres_ids and not actors_ids:
+        return Movie.objects.filter(genres__id__in=genres_ids)
+
+    if actors_ids and not genres_ids:
+        return Movie.objects.filter(actors__id__in=actors_ids)
```

Comment:
You can simplify your code
```suggestion
    queryset = Movie.objects.all()

    if genres_ids:
        queryset =  queryset.filter(genres__id__in=genres_ids)

    if actors_ids:
        queryset = queryset.filter(actors__id__in=actors_ids)
    return queryset
```